src/workflows/airqo_etl_utils/tests/test_airqo_data_drift_compute.py

```
@@ -0,0 +1,261 @@
+        ) as mock_baseline:
+            mock_baseline.return_value = baseline_params["baseline_id"]
+            baseline_id = AirQoDataDriftCompute.compute_baseline(
+                mock_df_raw,
+                baseline_params["device_id"],
+                baseline_params["pollutant"],
+                baseline_params["window_start"],
+                baseline_params["window_end"],
+                baseline_table,
+                region_min=baseline_params["region_min"]
+                if "region_min" in baseline_params
+                else baseline_params["region_min"],
```

Ruff (0.12.2)

Useless `if`-`else` condition

(RUF034)